banzai/images.py

```diff
@@ -16,10 +17,26 @@
 logger = logs.get_logger(__name__)
 
 
+class DataTable(object):
+    """
+    Object for storing astropy (or another table type) tables with an additional .name attribute which
+    determines the tables' extension name when it is saved as a fits file.
+    """
+    def __init__(self, data_table, name):
+        self.name = name
+        self._data_table = data_table
+
+    def __getitem__(self, item):
+        return self._data_table[item]
+
+    def __setitem__(self, key, value):
+        self._data_table[key] = value
+
+
 class Image(object):
 
-    def __init__(self, pipeline_context, filename=None, data=None, header=None,
-                 extension_headers=None, bpm=None):
+    def __init__(self, pipeline_context, filename=None, data=None, data_tables={},
```

Comment:
You should set `data_tables` to `None` and then to `{}` if it is `None`, as done with `header`

Reply:
fixed